docs/overview.rst

```diff
@@ -451,3 +451,51 @@ infrastructure, Linux and SSH. A good place to start is Heroku (https://www.hero
 We reference Heroku in our tutorials, but Agile Toolkit can work with any cloud
 hosting that runs PHP apps.
 
+
+Online-Demo showing many functions
+==================================
+
+There's a demo available of atk4/ui which shows many of the modules and functions available
+in atk4.
+You can watch & use the source code of each example to find best practice examples and to see
+how to use atk4 in certain application cases.
+
+You can find the demo here:
+https://agile-ui.readthedocs.io/en/latest/index.html
+
+
+Demo contained in the atk4 repository
```

Comment:
text below is quite specifically worded...

there should be:
- advise demos need clone/fork
- how sqlite database can be created: `php demos/_demo-data/create-sqlite-db.php`

Reply:
According to @georgehristov they are adviced to don't move the original repos and use them as they are.
What do we advice for consistency?  

1. clone just the "demos" directory or
2. clone the entire atk4/ui directory or
3. don't clone and use the files in /vendors/atk4/ui...

???

Reply:
@mvorisek, I added SQLITE but didn't commit. I'm waiting for feedback for the cloning issue...

Reply:
fork whole atk4/ui + composer install + init sqlite db, then make whole atk4/ui accessible by webserver

Reply:
What about moving just the "demos" directory in a subdirectory called "atk4/ui" on the webserver that's publicly available instead of the whole atk4/ui directory? 
Advantage: when updating atk4/ui, you have the demos separated and they aren't overwritten.

Reply:
impossible - you need to fork the whole ui

for separate vendor files, you need to understand composer more deeply and I think this should be not documented as too advanced for most users...

Reply:
adapted, please check